lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -803,10 +895,9 @@ def _set_scrape_job_spec(self, event):
         data.  In addition each of the consumer units also sets its own
         host address in Juju unit relation data.
 
-        Args:
-            event: a `CharmEvent` in response to which `MetricsEndpointProvider` will
-                forward scrape jobs, alert rules, metrics endpoint host addresses
-                and related metadata to the Prometheus charm.
+        All args are ignored. This is a commodity to enable this method to be
```

Comment:
Odd to say "all args are ignored" in an upstream function when they are ignored only downstream.
Can you instead ignore the event upstream instead of propagating it?